### ISSUE TITLE
Endless recusion in GetPitchBytes for ViewSubView

### ISSUE THREAD

**Issue by psychocoderHPC** [ComputationalRadiationPhysics/alpaka#136]
The trait specialization `GetPitchBytes` of `ViewSubView` results in an endless recursion.

The specialization calls the trait `getPitchBytes()` for the some input view which results in a call to the same method.

@BenjaminW3 Could you look over if I missed something

**Comment by psychocoderHPC** [comment]
Possible solution is to call the trait with the Buffer which is contained in `ViewSubView`.

```C++
return
                            mem::view::getPitchBytes<TIdx::value>(
                                view.m_Buf);
```

**Comment by BenjaminW3** [comment]
Yes, this solution seems to be correct. Please write a Pull Request.

**Comment by erikzenker** [comment]
Fixed by #143
